Maquetado/scss/tech.scss

```diff
@@ -0,0 +1,51 @@
+$color: #b2db59;
```

Comment:
This should be added to a colors.scss file inside a `variables` folder and import it in the application.scss at the begining so every file that comes after that one can use the colors declared